app/lib/shared/email.dart

```diff
@@ -116,6 +116,16 @@ class EmailMessage {
       throw ArgumentError('Invalid uuid: `$uuid`');
     }
   }
+
+  Map<String, Object?> toJson() {
+    return {
+      if (uuid != null) 'uuid': uuid,
```

Comment:
Wouldn't the field name `localMessageId` be more correct. This is:
https://en.wikipedia.org/wiki/Message-ID

-----------

Unrelated to this PR, but should we perhaps rename `EmailMessage.uuid` into `EmailMessage.localMessageId`?

Perhaps the docs for `localMessageId` should also link to https://www.jwz.org/doc/mid.html

Reply:
Renamed, added a TODO for a follow-up checking the recommendations on the link.